pkg/resiliency/types.go

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package resiliency
```

Comment:
This new package should probably be owned by some team in `CODEOWNERS`, maybe @cilium/sig-agent?

Reply:
Good catch!! Tx Joe!